doc/getting-started/building-from-source.rst

```diff
@@ -278,6 +278,8 @@ of the acrn-hypervisor directory):
    ``SCENARIO_FILE`` XML configuration files.  The ``TARGET_DIR`` parameter
    specifies what directory is used to  store configuration files imported
    from XML files. If the ``TARGED_DIR`` is not specified, the original
-   configuration files of acrn-hypervisor would be overridden.
+   configuration files of acrn-hypervisor would be overridden. Currentlly
```

Comment:
I'm not fond of putting comments like this in the documentation because we'll need to remember to remove them when the issue is fixed.  (Also, there's a typo on the line above for TARGET_DIR.)